src/SnD.Sdk.Storage.Minio/Providers/Configurations/MinioConfiguration.cs

```diff
@@ -0,0 +1,48 @@
+namespace Snd.Sdk.Storage.Minio.Providers.Configurations;
+
+/// <summary>
+/// Configuration settings for Minio Storage.
+/// </summary>
+public sealed class MinioConfiguration
+{
+    /// <summary>
+    ///  Minio S3 endpoint.
+    /// </summary>
+    public string Endpoint { get; set; }
+
+    /// <summary>
+    /// Access Key
+    /// </summary>
+    public string AccessKey { get; set; }
+
+    /// <summary>
+    ///  Secret Key
+    /// </summary>
+    public string SecretKey { get; set; }
+
+    /// <summary>
+    ///  Region.
+    /// </summary>
+    public string Region { get; set; }
+
+    /// <summary>
+    /// Enable SSL connectivity.
+    /// </summary>
+    public bool UseSsl { get; set; }
+
+    /// <summary>
+    /// Initialize from environment variables.
+    /// </summary>
+    /// <returns></returns>
+    public static MinioConfiguration CreateFromEnv()
+    {
+        return new MinioConfiguration
```

Comment:
use the `Environment.GetDomainEnvironmentVariable` method for env variables.

Reply:
`GetDomainEnvironmentVariable` is an extension method in `SnD.Sdk`. What would be the best approach to avoid creating dependencies? Should the common functionality be abstracted into another project?